--- et-audio.orig	2025-01-31 00:00:00.000000000 +0000
+++ et-audio	2025-01-31 00:00:00.000000000 +0000
@@ -85,6 +85,18 @@
 
           et-log "Applied amixer settings for ${ET_DEVICE_NAME}"
         ;;
+        "FT-710")
+          # Unmute Speaker a set volume. Adjust if remote station can't decode you. Your TX controls.
+          amixer -q -c ${AUDIO_CARD} sset Speaker Playback Switch 60% unmute
+          # Unmute Mic
+          amixer -q -c ${AUDIO_CARD} sset Mic Playback Switch 00% mute
+          # Set "L/R Capture" to 19. Adjust if you can't decode received audio. Your RX controls.
+          amixer -q -c ${AUDIO_CARD} sset Mic Capture Switch 60% unmute
+          # Disable Auto Gain Control
+          amixer -q -c ${AUDIO_CARD} sset 'Auto Gain Control' mute
+
+          et-log "Applied amixer settings for ${ET_DEVICE_NAME}"
+        ;;
         *)
           et-log "No ALSA configuration specified for this ET_DEVICE."
         ;;
